include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -3219,6 +3219,8 @@ struct OrtApi {
   * \param[in] cuda_options
   *
   * \snippet{doc} snippets.dox OrtStatus Return Value
+  *
+  * \version New in version 1.11.
```

Comment:
Since we never remove functions from the c_api, could this be `Available since version 1.11` or `Since Version 1.11`?

Reply:
> Since we never remove functions from the c_api, could this be `Available since version 1.11` or `Since Version 1.11`?

@gwang-msft  We do plan on removing functions in the c_api, but only for newer versions of the API, not older ones.